src/homepage/index.html

```diff
@@ -175,18 +175,20 @@ <h2 class="big-h2"><span class="fa fa-wrench"></span> <span>James is a <b>living
                   <b>Sources on Git:</b>
                   <br/>Read our sources from the <a href="git://git.apache.org/james-project.git" class="comm" alt="James Git sources">Apache Git</a>.<br/>
                   You can also come and interact with us on <a href="https://github.com/apache/james-project" class="comm" alt="James on GitHub">GitHub</a>.<br/>
-                  Here are some more instructions to <a href="/contribute.html" class="comm" alt="Contributing to James: guidelines">get starting contributing</a>.
                 </span>
               </li>
               <li class="post-template center-icon">
                 <a href="https://issues.apache.org/jira/issues/?jql=project%20%3D%20JAMES%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20newbie%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC" alt="contribute to the code"><span class="icon fa-cogs"></span></a>
                 <span class="details">
-                  <b>Contribute to the code:</b>
-                  <br/>Get started with code contribution! Search out the
+                  <b>Get started and contribute to this project:</b>
```

Comment:
Next time please add breakline otherwise it's a nightmare to review that one github !